src/user/introduction.rst

```diff
@@ -41,6 +41,8 @@ conda-forge is a community effort that tackles these issues:
  - By default, we build packages for macOS, Linux AMD64 and Windows AMD64.
  - Many packages are updated by multiple maintainers with an easy option to become a maintainer.
  - An active core developer team is trying to also maintain abandoned packages.
+ 
+You can refer to the glossary `here<https://conda-forge.org/docs/misc/00_intro.html#glossary>`.
```

Comment:
```suggestion
```

I don't think this is the place to put this reference

Reply:
Oops, this will break the CI I think @jakirkham. I don't think it's a valid RST ref.

Reply:
It didn't break but it doesn't render nicely. See last line in https://conda-forge.org/docs/user/introduction.html#why-conda-forge